bamboo/integration_tests/performance_tests/test_tools.py

```diff
@@ -93,52 +89,56 @@ def csv_to_dict(csv_path):
       if min_time > e['max_min_time']:
         errors.append('%.2f > %.2f %s %s max_min_time' % (min_time, e['max_min_time'], model_name, model_num))
+      numbers.append('%.2f %s %s min_time' % (min_time, model_name, model_num))
     for max_time in p['maxs']:
       if max_time > e['max_max_time']:
         errors.append('%.2f > %.2f %s %s max_max_time' % (max_time, e['max_max_time'], model_name, model_num))
+      numbers.append('%.2f %s %s max_time' % (max_time, model_name, model_num))
     for stdev in p['stdevs']:
       if stdev > e['max_stdev']:
         errors.append('%.2f > %.2f %s %s max_stdev' % (stdev, e['max_stdev'], model_name, model_num))
+      numbers.append('%.2f %s %s stdev' % (stdev, model_name, model_num))
     for accuracy in p['accuracies']:
       if accuracy < e['min_accuracy']:
         errors.append('%.2f < %.2f %s %s min_accuracy' % (accuracy, e['min_accuracy'], model_name, model_num))
-  print "Errors for: %s" % model_name
+      numbers.append('%.2f %s %s accuracy' % (accuracy, model_name, model_num))
+  print 'Errors for: %s (%d)' % (model_name, len(errors))
   for error in errors:
     print error
+  if log:
+    print 'All numbers for: %s (%d)' % (model_name, len(numbers))
+    for number in numbers:
+      print number
   assert errors == []
 
-def mnist_distributed_io_skeleton(executable, dir_name):
-  model_name = 'mnist_distributed_io'
+def lenet_mnist_skeleton(executable, dir_name, log=False):
+  model_name = 'lenet_mnist'
   output_file_name = '%s_output.txt' % model_name
   run_lbann(model_name=model_name, reader_name='mnist', optimizer_name='adagrad', output_file_name=output_file_name, executable=executable, dir_name=dir_name)
-  performance = get_performance(output_file_name)
-  run_tests(performance, model_name, dir_name)
+  performance = get_performance(output_file_name, log)
+  run_tests(performance, model_name, dir_name, log)
 
-def alexnet_skeleton(executable, dir_name):
+def alexnet_skeleton(executable, dir_name, log=False):
   model_name = 'alexnet'
   output_file_name = '%s_output.txt' % model_name
-  run_lbann(model_name=model_name, reader_name='imagenet', optimizer_name='adagrad', output_file_name=output_file_name, executable=executable, dir_name=dir_name)
-  performance = get_performance(output_file_name)
-  run_tests(performance, model_name, dir_name)
-
-def resnet50_skeleton(executable, dir_name):
-  model_name = 'resnet50'
-  output_file_name = '%s_output.txt' % model_name
-  run_lbann(model_name=model_name, reader_name='imagenet', optimizer_name='adagrad', output_file_name=output_file_name, executable=executable, dir_name=dir_name, num_epochs=1, procs_per_model=2)
-  performance = get_performance(output_file_name)
-  run_tests(performance, model_name, dir_name)
+  run_lbann(model_name=model_name, reader_name='imagenet', optimizer_name='adagrad', output_file_name=output_file_name, executable=executable, dir_name=dir_name, num_nodes=16, timeout=600, num_processes=32, procs_per_model=16)
```

Comment:
@timmoon10 Are these parameters correct? We're running alexnet on 16 nodes, with 32 processes and 16 processes per model (=> 32/16=2 models).